pkgs/servers/home-assistant/default.nix

```diff
@@ -21,6 +21,19 @@
 
 let
   defaultOverrides = [
+    # aiounify 29 breaks integration tests
+    (self: super: {
+      aiounifi = super.aiounifi.overridePythonAttrs (oldAttrs: rec {
+        version = "28";
```

Comment:
[29](https://github.com/home-assistant/core/blob/2021.11.0b3/requirements_all.txt#L258) will not be part of 2021.11.0. So it make sense to override it now.